meta-balena-common/recipes-core/packagegroups/packagegroup-resin.inc

```diff
@@ -20,4 +20,5 @@ RDEPENDS_${PN} = " \
     os-config \
     os-release \
     less \
+    development-features \
```

Comment:
Considering dropping the features part and making this just `development`

Reply:
I personally find `development features` more descriptive but either is fine with me.